src/persist-client/src/internal/metrics.rs

```diff
@@ -2326,6 +2328,14 @@ impl PushdownMetrics {
                 name: "mz_persist_pushdown_parts_inline_bytes",
                 help: "total size of parts not fetched because they were inline",
             )),
+            parts_faked_count: registry.register(metric!(
+                name: "mz_persist_pushdown_parts_faked_count",
+                help: "count of parts faked because of aggressive projection pushdown",
+            )),
+            parts_faked_bytes: registry.register(metric!(
+                name: "mz_persist_pushdown_parts_faked_bytes",
+                help: "total size of parts faked because of aggressive projection pushdown",
```

Comment:
```suggestion
                help: "total size of parts replaced with fakes by aggressive projection pushdown",
```

(To make it clearer that this isn't the size of the fake parts, which confused my very briefly.)

Reply:
Done!